### PR TITLE
notify user of bot's indexing progress

### PR DESCRIPTION
**Pull request by Huskydog9988**
This **should** work but I've yet to test it myself. Please do double check my work.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -324,6 +324,12 @@ async def index_channel(self, channel, discord_channel):
             if last_id and message_.id <= last_id:
                 break
 
+            if len(events) % 10000:
```
Comment (Liam-DeVoe) [inline main.py]: `if len(events) % 10000 == 0:`, right?

**Comment by Liam-DeVoe** [comment]
I was imaging this as continuously editing a single message for each channel, so the user doesn't get spammed. 

The "update message id" wouldn't have to be persisted to the database. Indexing is ephemeral at the moment anyway (restarting the bot in the middle will mess things up).

File: main.py (hunk 1 above)
Comment (Huskydog9988) [reply main.py]: yea

**Comment by Huskydog9988** [comment]
Yea that sounds like a better idea, i was just slapping something together to keep it simple

**Comment by Liam-DeVoe** [comment]
Thanks. I've changed the update frequency to 1k messages, since I think it's important to give immediate feedback to users, and 2.5k was a bit slow.

An ideal response rate might be something like "at 100, 200, 300, 400, 500, 1000, 2500 messages, and every 2500 thereafter". But that's both annoying to code and potentially even more confusing to users, so a flat 1k seems okay for now.

**Pull request merged**